global.yaml

```diff
@@ -443,9 +443,11 @@ global:
     enabled: false
     udsPath: ""
     # The JWT token for SDS and the aud field of such JWT. See RFC 7519, section 4.1.3.
-    # When a CSR is sent from Citadel Agent to the CA (e.g. Citadel), this aud is to make sure the
-    # JWT is intended for the CA.
     token:
+      # Filepath of the token.
+      path: ""
```

Comment:
Why empty ? The feature is enabled by sds.enabled, no need to have to configure the path too.